### DISCUSSION TITLE
kafka deployments not working with strimzi operator 0.34.0

### DISCUSSION THREAD

**Discussion by bharat15-ml** [strimzi/strimzi-kafka-operator#8283] [Q&A]
Hi All, 
I wanted to test kafka deployemnts on on-prem kubernetes cluster with strimzi operator 0.34.0 and with kafka version 3.4.0.  I am able to deploy strimzi operator-0.34.0 in on-prem k8s cluster but while trying kafka deployments not able to see any progress.

### I am trying this deployment on following k8s veriosn:-

Client Version: version.Info{Major:"1", Minor:"21", GitVersion:"v1.21.0", GitCommit:"cb303e613a121a29364f75cc67d3d580833a7479", GitTreeState:"clean", BuildDate:"2021-04-08T16:31:21Z", GoVersion:"go1.16.1", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"24", GitVersion:"v1.24.9-gke.100", GitCommit:"1ab6348c40ddb36445d5dc188b5e75a987b4fa0b", GitTreeState:"clean", BuildDate:"2022-12-08T21:48:12Z", GoVersion:"go1.18.9b7", Compiler:"gc", Platform:"linux/amd64"}

### helm version -
version.BuildInfo{Version:"v3.7.1", GitCommit:"1d11fcb5d3f3bf00dbe6fe31b8412839a96b3dc4", GitTreeState:"clean", GoVersion:"go1.16.9"}

After the strimzi operator- 0.34.0 deployment, I  tried below kafka deployment yaml-
kubectl create -f https://github.com/strimzi/strimzi-kafka-operator/blob/main/examples/kafka/kafka-persistent.yaml -n k1

I am not able to see any progess in the namespace k1. Below are error logs captured from strimzi operator pods.

Pls let me know on this issue. thanks!!

**Comment by scholzj** [comment]
Well, if you checked the log, then the problem seems quite obvious. You badly installed it and it is missing some RBAC rights. So you will need to doublecheck how you installed it, that you set up the RBAC rights correctly etc.

**Comment by bharat15-ml** [comment]
Thanks Scholzj, let me cross check on this.

**Reply by bharat15-ml** [reply]
Thanks @scholzj , I am able to resolve this issue. We can close this discussion.
